### DISCUSSION TITLE
MobiFlight Firmware on Raspberry Pi Pico with RP2040

### DISCUSSION THREAD

**Discussion by KejuBartin** [MobiFlight/MobiFlight-Connector#1245] [Ideas]
Would it be possible to make MobiFlight available on the Raspberry Pi Pico microcontroller? It does not have as many pins as an Arduino Mega but it is way cheaper. The firmware can be written in C/C++ and Python. Just an idea of mine, I don't know if it would make sense.

**Comment by elral** [comment]
Hi @KejuBartin, the Raspberry Pico is already supported. Just be careful that all pins are 3.3V and NOT 5V tolerant. Especially the I2C lines for the LCD need a level shifter.
Regards
Ralf

**Reply by KejuBartin** [reply]
Oh sorry, I did not know that it is already supported because it does not say that on the MobiFlight website:
https://www.mobiflight.com/en/index.html
![grafik](https://github.com/MobiFlight/MobiFlight-Connector/assets/88152801/6e67fc9f-ae07-4aea-aee5-e7e33364ea68)
